lib/utils.pm

```diff
@@ -12,6 +12,11 @@ sub get_log ($cmd, $name) {
     upload_logs($name) unless $ret;
 }
 
+sub install_packages($packages, $timeout = undef) {
+    $timeout //= 600;
+    assert_script_run(qq{retry -e -s 30 -r 7 -- sh -c "zypper -n --gpg-auto-import-keys ref && zypper --no-cd -n in $packages"}, $timeout);
```

Comment:
Let's see how well this works. I wasn't sure about `-n` and the timeout(s). Since this is racy by design I spawned 100 anyway: https://openqa.opensuse.org/tests/overview?build=poo%23158553

Reply:
Can you please open an issue on github.com/openSUSE/zypper/ and reference that here for our "multi-layer workaround"?